brd-ios/breadwallet/src/ApplicationController.swift

```diff
@@ -142,8 +142,8 @@ class ApplicationController: Subscriber, Trackable {
     func decideFlow() {
         UserManager.shared.refresh()
         if let nvc = rootNavigationController {
-            coordinator = BaseCoordinator(navigationController: nvc)
             coordinator?.modalPresenter = modalPresenter
+            coordinator = BaseCoordinator(navigationController: nvc)
```

Comment:
amm... this cannot work...
u cannot asing the modalPresenter to a nil (uninitialized) coordinator